modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/partials/navigation.hbs

```diff
@@ -111,6 +111,14 @@
                         {{/if}}
                     {{/each}}
                 </div>
+                <div id="tags-collapse" class="tags-collapse" align="center">
+                    <p class="small">
+                        <span id="tags-collapse-text">Show all tags&nbsp;&nbsp;</span>
```

Comment:
Ok, The correct reason can be found here : http://stackoverflow.com/questions/16951133/when-to-use-nbsp